source/common/grpc/common.cc

```diff
@@ -105,12 +105,14 @@ bool Common::resolveServiceAndMethod(const Http::HeaderEntry* path, std::string*
   if (path == nullptr || path->value().c_str() == nullptr) {
     return false;
   }
-  std::vector<std::string> parts = StringUtil::split(path->value().c_str(), '/');
+  auto parts = StringUtil::splitToken(path->value().c_str(), "/");
   if (parts.size() != 2) {
     return false;
   }
-  *service = std::move(parts[0]);
-  *method = std::move(parts[1]);
+  std::string service_str{parts[0]};
```

Comment:
nit: I would kill the local variables. You can assign a new string, or just `assign` directly into the output variables.